src/pages/workspaces/workspace/Dashboard/RightBoxSection.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { act, fireEvent, screen } from '@testing-library/react';
+import React from 'react';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+
+import { RightBoxSection } from './RightBoxSection';
+
+describe('RightBoxSection', () => {
+  it('renders without crashing', () => {
+    const { container } = render(<RightBoxSection title="Test Title" info="Test Info" persistenceId="testId" />);
+    expect(container).toBeInTheDocument();
```

Comment:
I would suggest ditching this test case, as it doesn't seem useful. If you are going to keep it, at least do an a11y check on the rendered component.